packages/dx-react-bootstrap4/jest.config.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path');
+
+module.exports = {
+  verbose: true,
+  transform: {
+    '^.+\\.(js|jsx)?$': 'babel-jest',
+  },
```

Comment:
Is `transform` required? [Note](https://jestjs.io/docs/en/getting-started#using-webpack)

> `babel-jest` is automatically installed when installing Jest and will automatically transform files if a babel configuration exists in your project.